### PR TITLE
2.4 version bump

### PR DESCRIPTION
**Pull request by rawbee**
## Description of the change

> Description here
## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 
### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: android/src/main/kotlin/com/getpinwheel/pinwheel/PinwheelPlugin.kt [hunk 1]
```diff
@@ -178,10 +200,7 @@ internal class NativeView(context: Context, messenger: BinaryMessenger, id: Int,
       token = ""
     }
 
-    webView = WebView(context)
-    webView?.let {
-      Pinwheel.init(it, readLinkToken(), pinwheelEventListener)
-    }
+    pinwheelView = PinwheelViewGroupManager.init(context, readLinkToken(), pinwheelEventListener, "react native", "3.0.0")
```
Comment (rawbee) [inline android/src/main/kotlin/com/getpinwheel/pinwheel/PinwheelPlugin.kt:203]: This needs to be changed to flutter, but I think flutter needs to be added to the allowlist. cc @roscioli

Comment (roscioli) [reply android/src/main/kotlin/com/getpinwheel/pinwheel/PinwheelPlugin.kt:203]: roger can implement in link

Comment (roscioli) [reply android/src/main/kotlin/com/getpinwheel/pinwheel/PinwheelPlugin.kt:203]: this is in production now for Link

File: lib/models.g.dart
[the remark this answers is not in the document]
Comment (rawbee) [reply lib/models.g.dart:1]: yep!

[the remark this answers is not in the document]
Comment (rawbee) [reply lib/models.g.dart:1]: same with `serializers.g.dart`

**Pull request merged**